src/test/resources/features/affordability_ssttpChanges/HappyPaths.feature

```diff
@@ -65,5 +65,33 @@ Feature: Happy Paths - Affordability
       | 0             | none     | 1           |
       | 1             | 0        | none        |
 
+### Affordability - Add Spending page
+
+  Scenario Outline: Validation on Add Income Page
```

Comment:
Should this be for Spending? not Income?

Reply:
Fixed